wp-content/plugins/jetpack/_inc/build/custom-css/custom-css/js/core-customizer-css.min.asset.php

```diff
@@ -0,0 +1 @@
+<?php return array('dependencies' => array(), 'version' => 'cb3d994c3cb34799d496d710b646dfdd');
```

Comment:
:no_entry_sign: **Error( severity 5 )**: When a multi-item array uses associative keys, each value should start on a new line (*WordPress.Arrays.ArrayDeclarationSpacing.AssociativeArrayFound*).